dll/win32/browseui/lang/pt-PT.rc

```diff
@@ -68,7 +73,7 @@ BEGIN
     BEGIN
         MENUITEM "&Mapear unidades de rede...", IDM_TOOLS_MAPNETWORKDRIVE
         MENUITEM "&Desligar unidade de rede...", IDM_TOOLS_DISCONNECTNETWORKDRIVE
-        MENUITEM "&Sincrinizar...", IDM_TOOLS_SYNCHRONIZE
+        MENUITEM "&Sincronizar...", IDM_TOOLS_SYNCHRONIZE
```

Comment:
This is good 1/2. Aside those 2 changed lines, I can't spot anything that would really improve the translation.